core/controllers/contributor_dashboard_admin_test.py

```diff
@@ -739,10 +739,23 @@ def setUp(self) -> None:
         user_services.add_user_role(
             self.contributor_id, feconf.ROLE_ID_TRANSLATION_ADMIN)
 
+        user_settings_1 = user_services.create_new_user(
```

Comment:
Can you rename these variables to have proper meanings?

Reply:
Fixed. PTAL again.